src/gentypes.jl

```diff
@@ -340,7 +346,7 @@ end
 function _importexprs(mod::ROSMsgModule)
     imports = Expr[Expr(:import, :RobotOS, :AbstractMsg)]
     othermods = filter(d -> d != _name(mod), mod.deps)
-    append!(imports, [Expr(:using,Symbol(m),:msg) for m in othermods])
+    append!(imports, [Expr(:using,:Main,Symbol(m),:msg) for m in othermods])
```

Comment:
See https://github.com/JuliaLang/julia/issues/17997 and https://github.com/JuliaLang/julia/pull/23579.